pkgs/applications/version-management/gitkraken/default.nix

```diff
@@ -166,7 +166,9 @@ let
 
     nativeBuildInputs = [
       copyDesktopItems
-      (wrapGAppsHook3.override { makeWrapper = makeShellWrapper; })
+      # override doesn't preserve splicing https://github.com/NixOS/nixpkgs/issues/132651
+      # Has to inherit from `buildPackages` even though `makeWrapper` is spliced because `propagatedBuildInputs` would pick the wrong one because of a different offset.
+      (buildPackages.wrapGAppsHook3.override { inherit (buildPackages) makeWrapper; })
```

Comment:
What about makeShellWrapper? Or is the change intentional?

Reply:
`makeWrapper` is `makeShellWrapper` so I just used that

I now switched them to ` makeWrapper = buildPackages.makeShellWrapper; ` so if `makeWrapper` is switched to the binary wrapper the migration will be a bit easier.